wire-ios/Wire-iOS/Resources/ar.lproj/Localizable.strings

```diff
@@ -1477,7 +1477,6 @@
 
 // Generic
 "registration.no_history.restore_backup_failed.title" = "حدث خطأ";
-"registration.no_history.restore_backup_failed.message" = "لا يمكن استعادة تاريخك.";
```

Comment:
or we just add %@ at the back, here and in all other localisations

Reply:
can't do just add %@, as I guess here it's right to left language